keps/sig-storage/2268-non-graceful-shutdown/README.md

```diff
@@ -386,18 +384,35 @@ logs or events for this purpose.
   The usage of this feature requires the manual step of applying a taint
   so the operator should be the one applying it.
 
+###### How can someone using this feature know that it is working for their instance?
+
+<!--
+For instance, if this is a pod-related feature, it should be possible to determine if the feature is functioning properly
+for each individual pod.
+Pick one more of these and delete the rest.
+Please describe all items visible to end users below with sufficient detail so that they can verify correct enablement
+and operation of this feature.
+Recall that end users cannot usually observe component logs or access metrics.
+-->
+
+- [X] API .status
+  If it works, pods in the stateful workload should be re-scheduled to another
+  running node. `Phase` in Pod `Status` should be `Running` for a new Pod
+  on the other running node.
+  If not, check the pod status to see why it does not come up.
+
 ###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
 
 <!--
 Pick one more of these and delete the rest.
 -->
   - [X] Metrics
     - Metric name: 
-      - We can add new metrics `deleting_pods_total`, `deleting_pods_error_total`
+      - We added new metrics `deleting_pods_total`, `deleting_pods_error_total`
```

Comment:
```suggestion
      - We added new metrics `deleting_pods_total` which is number of pods that are being forcefully deleted since the Pod GC Controller started, `deleting_pods_error_total`
```

Reply:
Could you please show me URL of doc or PullRequest about `deleting_pods_error_total` ?

Reply:
@YuikoTakada https://github.com/kubernetes/kubernetes/pull/113519/files#diff-44ad977fee31d8981729c0cf78c9ad9917dee55958e544c6748109ea61bd64f5R40

Reply:
```
`deleting_pods_error_total` which is number of errors encountered when forcefully deleting the pods since the Pod GC Controller started.
```

Reply:
Done